benchmarking/reconstruct.py

```diff
@@ -61,7 +61,54 @@ def main(phantom, max_iter, output_dir, ncore, parameters):
     data = np.load(os.path.join(output_dir, phantom, 'simulated_data.npz'))
     dynamic_range = np.max(data['original'])
     if parameters is not None:
-        parameters = ast.literal_eval(parameters)
+        if parameters == 'gridrec':
```

Comment:
Do not break the old API. Add a new option instead of changing the functionality of the existing `--parameters` option. Remember to update the docstring! to indicate how to use the new option.

Reply:
What do you mean by docstring?

Reply:
I meant the string which contains documentation. i.e. the string that immediately follows a function declaration and the `help` parameter for the `@click.option` decorators.